gdal/scripts/vagrant/gdal.sh

```diff
@@ -18,15 +18,29 @@ echo rsync -a /vagrant/gdal/ /home/vagrant/gnumake-build-gcc4.8/ > /home/vagrant
 echo rsync -a /vagrant/autotest/ /home/vagrant/gnumake-build-gcc4.8/autotest >> /home/vagrant/gnumake-build-gcc4.8/resync.sh
 
 chmod +x /home/vagrant/gnumake-build-gcc4.8/resync.sh
-cd gnumake-build-gcc4.8
+cd /home/vagrant/gnumake-build-gcc4.8
 
-#  --with-ecw=/usr/local --with-mrsid=/usr/local --with-mrsid-lidar=/usr/local --with-fgdb=/usr/local
+export CCACHE_CPP2=yes
+
+ARCH_FLAGS=""
+AVX2_AVAIL=1
+grep avx2 /proc/cpuinfo >/dev/null || AVX2_AVAIL=0
+if [[ "${AVX2_AVAIL}" == "1" ]]; then
+        ARCH_FLAGS="-mavx2"
+        echo "AVX2 available on CPU"
+else
+        echo "AVX2 not available on CPU."
+        grep flags /proc/cpuinfo | head -n 1
+fi
+
+CFLAGS=$ARCH_FLAGS CXXFLAGS=$ARCH_FLAGS CC="ccache clang" CXX="ccache clang" LDFLAGS="-lstdc++" \
```

Comment:
a bit weird to use clang when the directory name is gnumake-build-gcc4.8

Reply:
indeed! Have switched to gcc for now. ✅